search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/ext/GenerativeQAParameters.java

```diff
@@ -168,6 +207,7 @@ public GenerativeQAParameters(StreamInput input) throws IOException {
         this.interactionSize = input.readInt();
         this.timeout = input.readInt();
         this.llmResponseField = input.readOptionalString();
+        this.llmMessages.addAll(input.readList(MessageBlock::new));
```

Comment:
You are introducing a new field into bytestream here, which needs to do a version control here to prevent bwc failing